init.sls

```diff
@@ -15,7 +15,7 @@ docker:
 
 # Deploy daemon.json if options are given in pillar.
 # Make sure it's not existent if not.
-{% if salt['pillar.get']('docker:daemon', undefined) is defined %}
```

Comment:
With `default` not defined, non-existing pillars will return an empty string, see [docs](https://docs.saltproject.io/en/latest/ref/modules/all/salt.modules.pillar.html#salt.modules.pillar.get).

I think we should probably explicitly return `None` as a default.